sklearn/linear_model/_glm/glm.py

```diff
@@ -448,6 +452,7 @@ class PoissonRegressor(GeneralizedLinearRegressor):
     >>> clf.predict([[1, 1], [3, 4]])
     array([10.676..., 21.875...])
     """
+    @_deprecate_positional_args
```

Comment:
I haven't checked for the rest, but I believe these estimators were introduced with a `*` right from the beginning. That is, for them we don't need the deprecation that the decorator gives us. Could you also double-check the rest of the changes here? Thanks!

Reply:
You are absolutely right @NicolasHug, these estimators were introduced with the keyword arguments (`*`) from the beginning.